docs/alerts.md

```diff
@@ -262,3 +262,51 @@ handler2 = (tables=<-) => slack.notify(channel: "#flux")
 tables |> handler1() |> to(bucket: "notifications")
 tables |> handler2() |> to(bucket: "notifications")
 ```
+
+### Tasks
+
+Using the above, this can be integrated with tasks fairly easily and meet the above requirements.
+
+This might be the check script.
+
+```
+// Injected.
+option task = {
+	every: 1m,
+}
+
+// Possibly an option like now?
+notify = (tables=<-) => tables |> to(bucket: "system")
+
+// User provided.
+from(bucket: "telegraf")
+    |> range(start: -task.every)
+    |> filter(fn: (r) => r._measurement == "cpu" and r._field == "usage_system")
+    |> check(
+	crit: (r) => r._value > 90,
+    )
+    |> notify()
```

Comment:
This design doesn't satisfy the requirement that by default check statuses are written to a bucket. A power user using the `check` method would need to remember to write to the system bucket in every alert script.

Reply:
Yes, it does. We would insert the implementation of `notify()` to write to the system bucket. There's no necessity for them to write to the system bucket because that's what `notify()` does.

Reply:
See my comment above, the check function or whatever wrapper around it needs to persist the status to a bucket without the need to call a notify function.